src/cms/static/js/tinymce_plugins/autolink_phone/plugin.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ *
+ * Version: 5.7.0 (2021-02-10)
+ */
```

Comment:
Do you have a source for this script? The copyright notice suggests that this is an official plugin? Is there a possibility to install it somehow without having to track it in the vcs?

Reply:
That's the official autolink plugin, but as it doesn't support the phone-detection I came up with a own version, which only searches for phone numbers. I don't think, that we can expand this by a patch, as we need to provide some serious customization, which includes:
- Adding additional RegEx
- Adding custom Links
- (Most important) We need to change the window-offset, which is used in multiple spots in the plugin. We do need to change this to identify numbers with only 3 digits.